app/models/imageLayer.js

```diff
@@ -0,0 +1,8 @@
+function ImageLayerViewModel(data) {
+  this.Id = data.Id;
+  this.Created = data.Created;
+  this.CreatedBy = data.CreatedBy.replace('/bin/sh -c #(nop) ','').replace('/bin/sh -c ', 'RUN ');
```

Comment:
I'd rather see a filter in the view instead of replacing the data directly in the model.

Keep it this way in the model: `this.CreatedBy = data.CreatedBy;` and create a new `imageLayer` filter in `filter.js` to replace the unwanted strings.